src/clientsettingsdlg.cpp

```diff
@@ -53,6 +53,11 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient* pNCliP, CClientSettings* pNSet
     layout()->setMenuBar ( pMenu );
 #endif
```

Comment:
Smells like avoidable duplication.

Reply:
Do we live with this for now as it's unrelated?

Reply:
Unrelated, but worth an issue